app/models/container_deployment.rb

```diff
@@ -0,0 +1,11 @@
+class ContainerDeployment < ApplicationRecord
+  has_many :container_deployment_authentications
+  belongs_to :deployed_ems, :class_name => 'ExtManagementSystem'
+  belongs_to :deployed_on_ems, :class_name => 'ExtManagementSystem'
```

Comment:
Sure, so the deployed_ems is the openshift/atomic ems that you are creating by using the deployment feature, the deployed_on_ems is either the provider you provision vms on and then deploy the cluster or the provider that you choose existing vms to deploy on.